modules/CodeTablesAdmin/src/client/AbstractCodeTablesAdminView.html

```diff
@@ -0,0 +1,125 @@
+<script type="text/template" id="AbstractCodeTablesAdminView" xmlns="http://www.w3.org/1999/html">
+    <div class="form-horizontal">
+        <div class="control-group">
+            <label class="control-label bv_group_codeTablesAdminCode"><%=upperDisplayName%> Code</label>
```

Comment:
Should this have an asterisk too? When I played with the UI it seemed Code and Name were both required, but only Name had the * next to it

Reply:
Thanks, yes, I made it required, i'll add the asterisk